src/packages/client/src/__tests__/integration/errors/connection-limit/test.ts

```diff
@@ -0,0 +1,34 @@
+import { getTestClient } from '../../../../utils/getTestClient'
+
+describe('connection-limit', () => {
+  expect.assertions(1) // cannot have any clients currently connected to db.
+  let clients: Array<any> = []
+
+  afterAll(async () => {
+    for (const client of clients) {
+      await client.$disconnect()
```

Comment:
You can try to use `Promise.all` here instead to speeds things up. (But not urgent)
Something like this:
```ts
await Promise.all(clients.map(c => c.disconnect())
```
`Promise.all` runs everything in parallel, whereas `for of` runs things in sequence.